tests/migration/namespace.go

```diff
@@ -0,0 +1,391 @@
+			Expect(err).ToNot(HaveOccurred())
+			libstorage.EventuallyDV(sourceDV, 240, Or(matcher.HaveSucceeded(), matcher.WaitForFirstConsumer()))
+
+			sourceVM = createAndStartVMFromVMISpec(sourceVMI)
+			deviceName := ""
+			Eventually(func() string {
+				sourceVMI, err := virtClient.VirtualMachineInstance(sourceVMI.Namespace).Get(context.Background(), sourceVMI.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				for _, v := range sourceVMI.Status.VolumeStatus {
+					if v.Name == "disk1" {
+						deviceName = v.Target
+						return v.Target
+					}
+				}
+				return ""
+			}).WithTimeout(60 * time.Second).WithPolling(2 * time.Second).ShouldNot(BeEmpty())
+
+			for _, volume := range sourceVMI.Status.VolumeStatus {
+				if volume.Name == "disk1" {
+					deviceName = volume.Target
+				}
+			}
+			By("Writing data to extra disk")
+			Expect(console.LoginToCirros(sourceVMI)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, fmt.Sprintf("sudo mkfs.ext4 /dev/%s", deviceName), 30*time.Second)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, "mkdir test", 30*time.Second)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, fmt.Sprintf("sudo mount -t ext4 /dev/%s /home/cirros/test", deviceName), 30*time.Second)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, "sudo chmod 777 /home/cirros/test", 30*time.Second)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, "sudo chown cirros:cirros /home/cirros/test", 30*time.Second)).To(Succeed())
+			Expect(console.RunCommand(sourceVMI, "printf 'important data' &> /home/cirros/test/data.txt", 30*time.Second)).To(Succeed())
+
+			targetDV, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(targetDV)).Create(context.Background(), targetDV, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			libstorage.EventuallyDV(targetDV, 240, Or(matcher.HaveSucceeded(), matcher.WaitForFirstConsumer()))
+
+			targetVM = createReceiverVMFromVMISpec(targetVMI)
+			sourceMigration := libmigration.NewSource(sourceVMI.Name, sourceVMI.Namespace, migrationID, connectionURL)
+			targetMigration := libmigration.NewTarget(targetVMI.Name, targetVMI.Namespace, migrationID)
+			sourceMigration, targetMigration = libmigration.RunDecentralizedMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, sourceMigration, targetMigration)
+			libmigration.ConfirmVMIPostMigration(virtClient, targetVMI, targetMigration)
+			By("Verifying data on extra disk")
+			Eventually(func() string {
+				targetVMI, err := virtClient.VirtualMachineInstance(targetVMI.Namespace).Get(context.Background(), targetVMI.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				for _, v := range targetVMI.Status.VolumeStatus {
+					if v.Name == "disk1" {
+						deviceName = v.Target
+						return v.Target
+					}
+				}
+				return ""
+			}).WithTimeout(60 * time.Second).WithPolling(2 * time.Second).ShouldNot(BeEmpty())
```

Comment:
Nit
```suggestion
			}).WithTimeout(1 * time.Minute).WithPolling(2 * time.Second).ShouldNot(BeEmpty())
```
Few more occurrences in this file.